2019/d01p1FuelCounterUpper.d

```diff
@@ -0,0 +1,7 @@
+unittest {
```

Comment:
You say you've been practicing. Demonstrate this with a show and tell. Help the audience understand why you find this D so attractive.

Reply:
I submit that any programmer can read this and understand what is going on. I further submit that this code _as a solution to the AoC problem_ can be easily explained to a non-programmer.

This line alone says so much! Everything enclosed in the braces is intended to prove something meaningful. This has to be my favourite feature of D so far. I like to "sketch" in code so having a code-based canvas to draw on helps me interpret the problem I'm working on. Like here.

Reply:
I submit that any programmer can read this and understand what is going on. I further submit that this code _as a solution to the AoC problem_ can be easily explained to a non-programmer.

This line alone says so much! Everything enclosed in the braces is intended to prove something meaningful. This has to be my favourite feature of D so far. I like to "sketch" in code so having a code-based canvas to draw on helps me interpret the problem I'm working on. Like here.